app/models/user.rb

```diff
@@ -628,6 +635,7 @@ def tag_keywords_for_search
     "#{employer_name}#{mostly_work_with}#{available_for}"
   end
 
+  # TODO: this can be removed once we migrate away from ES
   def search_score
```

Comment:
`search_score` is used by Elasticsearch to compute `hotness_score` for a user. I moved that to be calculated on the fly during sorting when needed. That's the only place where we actually use it